reana_job_controller/config.py

```diff
@@ -51,7 +51,7 @@
 ).split(",")
 """List of supported compute backends provided as docker build arg."""
 
-KRB5_CONTAINER_IMAGE = os.getenv("KRB5_CONTAINER_IMAGE", "reanahub/krb5:latest")
+KRB5_CONTAINER_IMAGE = os.getenv("KRB5_CONTAINER_IMAGE", "reanahub/reana-auth-krb5:latest")
```

Comment:
Let's wait with merging this change once we have full `reana-auth-krb5` image building ready in Kind?  

We also need to push the image to Docker Hub; currently only `krb5:latest` is present there.

Reply:
(BTW shall we aim at using 1.0 release here? Well, we can attack this at the time of amending Kind to have release builds.)

Reply:
Since elsewhere we use `v1.0.0` (e.g. in reana-env-jupyter, or in examples, or in REANA components itself, please use 1.0.0 here.

Also, please rebase against latest master.

Otherwise it works OK for me with locally build r-a-krb5 image. (We can release on Docker Hub after merging the other r-a-krb5 PR.)